cartography/intel/aws/securityhub.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_hub(boto3_session: boto3.session.Session) -> Dict:
+    client = boto3_session.client('securityhub')
+    try:
+        return client.describe_hub()
+    except client.exceptions.ResourceNotFoundException:
```

Comment:
Maybe we should log a warning here too? Up to you

Reply:
I think it'll throw an exception if it's not enabled for the account, so I think this is what we want.